src/subcommand/server.rs

```diff
@@ -492,6 +493,29 @@ impl Server {
     Redirect::to(&format!("/sat/{sat}"))
   }
 
+  async fn sat_from_n(
+    Extension(index): Extension<Arc<Index>>,
+    Path((DeserializeFromStr(sat), DeserializeFromStr(n))): Path<(
+      DeserializeFromStr<Sat>,
+      DeserializeFromStr<i64>,
+    )>,
+  ) -> ServerResult<Json<SatId>> {
+    let mut inscriptions = index.get_inscription_ids_by_sat(sat)?;
```

Comment:
This function is very inefficient at the moment because it loads all inscriptions on a sat and goes to multiple tables to order the multimap result. Prob have to wait for #2664 to make this efficient